### ISSUE TITLE
Crime Rate Prediction

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#620]
Define You:

- [ ] KWOC 2023 Participant 
- [ ] Contributor

Hey OS participants, good to see you.

This project will help us in predicting the crime rate around various states with various parameters taken into consideration.

Dataset : https://www.kaggle.com/nehaprabhavalkar/crime-in-india

If you like to work on this issue, follow the given guidelines for code as well as for the README. You can try to compare it using different algorithms for better modelling practices.

**Comment by adi271001** [comment]
please assign this to me @prathimacode-hub

**Comment by prathimacode-hub** [comment]
If you're interested to work under DWOC, let me know? @adi271001

**Comment by adi271001** [comment]
Sure

**Comment by dp2426** [comment]
Hello @prathimacode-hub please assign this to me. I would like to contribute on this under CodePeak 2025.

**Comment by prathimacode-hub** [comment]
Issue assigned to @dp2426

**Comment by dp2426** [comment]
I explored the domain, and found the independent variables to predict the target. I am cleaning the data. Next going to compare multiple models. Can you give me advice what else to do next?

**Comment by prathimacode-hub** [comment]
You have to follow the complete data science lifecycle. Should i mention specifically? @dp2426

**Comment by dp2426** [comment]
Yes! If you can it would definitely help me to verify at what stage I have reached! Thanks!

**Comment by prathimacode-hub** [comment]
Data Preparation, Data Processing, Data Analysis, Data Visualization, 3-4 Algorithm Models that will fit problem statement with accuracies, finally summarizing statement of which algorithm give best optimal result and accuracy. These is an overall data science lifecycle. @dp2426

**Comment by prathimacode-hub** [comment]
Since there is no update of PR from your end, I'm unassigning you. As well, CodePeak 2025 program is wrapped up. All the best for your future endeavours.. @dp2426
